pkg/broker/broker.go

```diff
@@ -927,20 +1021,20 @@ func (a AnsibleBroker) Bind(instance bundle.ServiceInstance, bindingUUID uuid.UU
 			// unknown error
 			case err != nil && !a.dao.IsNotFoundError(err):
 				return nil, false, err
-			// If there is a job in "succeeded" state, or no job at all, or
-			// the referenced job no longer exists (we assume it got
-			// cleaned up eventually), assume everything is complete.
+				// If there is a job in "succeeded" state, or no job at all, or
```

Comment:
I think that this should be inline with the case statement

Reply:
I think that too, I think my IDE changed that while formatting code, will re-change that and hate ide just a little bit...